### DISCUSSION TITLE
Add in more species

### DISCUSSION THREAD

**Discussion by BrittanyBunk** [falling-fruit/falling-fruit-web#935] [Ideas]
I know this is a low-priority item, but I'd like to write my ideas here anyway - for the off-chance they get worked on one day.

There's a lot of species that aren't mentioned in the list, and adding them would help people know more about what can be foraged than what they think of. Here's what can be added:
- crepe myrtle
- cypress
- wine grapes
- wild mustard
- snapdragon
- carrotwood
- canary island pine

maybe others have ones they'd like to add to this list of labels not on the map officially?

**Comment by ezwelty** [comment]
A recurring point of confusion is that a species may exist in the database but not be displayed on https://fallingfruit.org because it was not considered of sufficient food interest to be placed in the `forager` category. https://beta.fallingfruit.org addresses this by making all species accessible to users. From your list, this applies to:

- Crape myrtle (_Lagerstroemia indica_)
- Cypress (_Cupressus_)
- Canary Island pine (_Pinus canariensis_)

I'm not too keen about pre-populating the database with types that may or may not be used some day. Currently, new types are introduced incrementally by users when they do not find what they are looking for in the list. These "pending types" are then reviewed (sporadically, when I have time) and either merged with an existing type, cleaned up as needed and published as a new official type, or left unresolved because they are too ambiguous. The beta website adds the ability to filter locations by these pending types, and assign pending types to locations, which should help address the frustration of users not being able to make use of the types they just submitted.

For the others in your list:

- "wine grapes": Would be matched to existing _Vitis vinifera_.
- "wild mustard" is ambiguous: To be matched to existing _Brassica_ genus (based on additional info in the location description).
- "snapdragon": To be added from pending type of same name. I presume you mean species in the _Antirrhinum_ genus?
- "carrotwood": To be added from pending type of same name as _Steganotaenia araliacea_ based on additional info in the location description.

Note: The existing feature of suggesting a new type is still missing but is planned for the beta website: https://github.com/falling-fruit/falling-fruit-web/issues/305.

**Comment by ezwelty** [comment]
The beta website now supports suggesting a new type. https://beta.fallingfruit.org
